appserver/web/web-core/pom.xml

```diff
@@ -127,6 +127,18 @@
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-engine</artifactId>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
```

Comment:
Project uses easymock already, could it be used instead?

Reply:
Changed to use easymock! Please check it again.